src/js/controllers/tab-receive.js

```diff
@@ -74,6 +74,12 @@ angular.module('copayApp.controllers').controller('tabReceiveController', functi
           paymentSubscriptionObj.addr = $scope.addr
       }
 
+      try {
+        clipboardService.copyToClipboard($scope.protocolHandler + ":" + $scope.addr);
```

Comment:
I am still facing the following issues:
1) For BCH, while Legacy is selected, when generating address, it copies the CashAddr format. It should copy the Legacy format.
2) For BTC, when generating address, the copied text contains "bitcoin:" prefix. This should be removed as it is only used for a payment request URI scheme BIP21. The copied text should only contain the address, e.g. 1JH4u2GaQTu7pWXMiGBeXr6oAbWmVVkpMS. This should also apply to the Legacy BCH address comment in item 1, where no prefixes should be added for the legacy address.